cg/constants/observations.py

```diff
@@ -67,6 +77,23 @@ class MipDNAObservationsAnalysisTag(StrEnum):
     FAMILY_PED: str = "pedigree"
 
 
+class NalloObservationsAnalysisTag(StrEnum):
+    """Nallo observations files analysis tags."""
+
+    SNV_VCF: str = "vcf-snv"
+    SV_VCF: str = "vcf-sv"
+    FAMILY_PED: str = "pedigree"
```

Comment:
This is kind of a new rule we have agreed on sysdev. Nothing really crucial, it is just more correct and fixes some warnings
```suggestion
    SNV_VCF = "vcf-snv"
    SV_VCF = "vcf-sv"
    FAMILY_PED = "pedigree"
```